ale_linters/swift/appleswiftformat.vim

```diff
@@ -0,0 +1,69 @@
+" Author: Klaas Pieter Annema <https://github.com/klaaspieter>
+" Description: Support for swift-format https://github.com/apple/swift-format
+
+let s:default_executable = 'swift-format'
+call ale#Set('swift_appleswiftformat_executable', s:default_executable)
+call ale#Set('swift_appleswiftformat_use_swiftpm', 0)
+
+function! ale_linters#swift#appleswiftformat#GetExecutable(buffer) abort
```

Comment:
This function would look way better by removing all the unnecessary else statements:

```vimscript
function! ale_linters#swift#appleswiftformat#GetExecutable(buffer) abort
    if !ale#Var(a:buffer, 'swift_apple_swift_format_use_swiftpm')
        return ale#Var(a:buffer, 'swift_apple_swift_format_executable')
    fi
    
    if ale#path#FindNearestFile(a:buffer, 'Package.swift') is# ''
        " If there is no Package.swift file, we don't use swift-format even if it exists,
        " so we return '' to indicate failure.
        return ''
    fi

    return 'swift'
endfunction
```